### PR TITLE
Add support multiple business entities

### PR DESCRIPTION
**Pull request by Patrick-Duvall**
JIRA: https://jira.recurly.net/browse/BEN-295

Adds support for multiple business entities

### CODE REVIEW

File: recurly/__init__.py [hunk 1]
```diff
@@ -470,6 +470,40 @@ class BillingInfoFraudInfo(recurly.Resource):
         'decision',
     )
 
+class BusinessEntity(Resource):
```
Comment (andcoca) [inline recurly/__init__.py:474]: What do you think of adding `override_business_entity_id` to `class Account(Resource)` attributes to support the creation/update of an account with an `override_business_entity`.

Comment (Patrick-Duvall) [reply recurly/__init__.py:474]: Good catch

File: tests/fixtures/business_entity/list.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Authorization: Basic YXBpa2V5Og==
+User-Agent: {user-agent}
+
+
+HTTP/1.1 200 OK
+Content-Type: application/xml; charset=utf-8
+
+<?xml version="1.0" encoding="UTF-8"?>
+<business_entities type="array">
+    <business_entity href="https://pcc.recurly.dev:3000/v2/business_entities/sy8yqpgxmeqc">
```
Comment (andcoca) [inline tests/fixtures/business_entity/list.xml:13]: Just add an `<invoices>` node for these xml schemas.

File: tests/test_resources.py [hunk 3]
```diff
@@ -3168,6 +3169,61 @@ def test_get_external_product(self):
...
 
+    
+    def test_get_business_entity(self):
+        with self.mock_request('business_entity/get.xml'):
+            entity = BusinessEntity.get('sy8yqpgxmeqc')
+
+        self.assertEqual(entity.id, 'sy8yqpgxmeqc')
+        self.assertEqual(entity.code, '302-test')
+        self.assertEqual(entity.name, '302-test')
+        self.assertEqual(entity.name, '302-test')
```
Comment (andcoca) [inline tests/test_resources.py]: Just get rid of this, it's duplicated ^.

**Review by andcoca — APPROVED** [review]

**Pull request merged**